include/sdbus-c++/IConnection.h

```diff
@@ -133,6 +135,25 @@ namespace sdbus {
          */
         virtual std::string getUniqueName() const = 0;
 
+        /*!
+         * @brief Attach the bus connection to an sd-event event loop
+         *
+         * @throws sdbus::Error in case of failure
+         */
+        virtual void attachSdEventLoop(sd_event *ev, int priority = 0) = 0;
```

Comment:
These new virtual methods shall be moved all the way down in the class, just after `addMatch()` and before deprecated `enterProcessingLoop()`, to not break ABI compatibility, and to not have to create a new major version...